### PR TITLE
Restore PAPI v1 stdout/stderr log streaming.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala [hunk 1]
```diff
@@ -343,11 +349,16 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
...
+        |mkfifo "$$$out" "$$$err"
+        |trap 'rm "$$$out" "$$$err"' EXIT
+        |tee $stdoutRedirection < "$$$out" &
+        |tee $stderrRedirection < "$$$err" >&2 &
         |(
         |cd $cwd
         |ENVIRONMENT_VARIABLES
         |INSTANTIATED_COMMAND
-        |) $stdinRedirection > $stdoutRedirection 2> $stderrRedirection
+        |) $stdinRedirection > "$$$out" 2> "$$$err"
```
Comment (cjllanwarne) [inline backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: So many "💲💲"/ "💲💲💲"/ "💲💲💲💲"s!!

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala [hunk 2]
```diff
@@ -322,14 +322,20 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
...
       s"ln -s $cwd $d"
     } getOrElse ""
 
     // Only adjust the temporary directory permissions if this is executing under Docker.
     val tmpDirPermissionsAdjustment = if (isDockerRun) s"""chmod 777 "$$tmpDir"""" else ""
-    
+
+    // The `tee` trickery below is to be able to redirect to known filenames for CWL while also streaming
+    // stdout and stderr for PAPI to periodically upload to cloud storage.
+    // https://stackoverflow.com/questions/692000/how-do-i-write-stderr-to-a-file-while-using-tee-with-a-pipe
```
Comment (cjllanwarne) [inline backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: Ah, "tea with a pipe"... those were the days!

**Comment by mcovarr** [comment]
stdout/stderr filenames shouldn't be changing, there's just a layer of indirection with the FIFO.

File: backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala (hunk 1 above)
Comment (mcovarr) [reply backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala]: This code is money.

**Pull request merged**